app/assets/javascripts/float-kit/addon/lib/update-position.js

```diff
@@ -63,17 +48,38 @@ export async function updatePosition(trigger, content, options) {
       middleware.push(inline());
     }
 
-    middleware.push(visibilityOptimizerMiddleware);
+    const visibilityOptimizer =
```

Comment:
Need to move this down inside the `if` check, since it's not used otherwise

Reply:
actually it is used, cause visibility optimiser none is checked first.